server/models/Interface.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const { getNewestRedditPosts } = require("./../routes/reddit");
+
+const interface = class Interface {
+  constructor() {
+    this.mentions = []
```

Comment:
It should save state actually. I'll give you more details on this feature today during our standup.

Reply:
PS: Remember to keep it one feature per PR. I would approve the css changes, but I can't because it's in the same PR as the interface, which isn't ready yet.

Reply:
Alright, I reverted the css changes for the navbar and should be ready to merge